src/test/java/de/orchound/gameoflife/painting/FillPainterTest.java

```diff
@@ -0,0 +1,55 @@
+package de.orchound.gameoflife.painting;
+
+import de.orchound.gameoflife.model.Game;
+import org.joml.Vector2i;
+import org.joml.Vector2ic;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class FillPainterTest {
+	Game game = new Game(6, 6);
+	FillPainter fillPainter = new FillPainter(game);
+
+	@BeforeEach
+	void setUp() {
+		game.clear();
+	}
+
+	@Test
+	void testPaint() {
```

Comment:
Please implement two test methods `testFillBoard()` and `testFillSquare()`.
The former simply fill paints any cell of an empty board and verifies that the entire board is set.
The latter could paint a 4x4 square on the 6x6 board and then fill paint that square. (Then verifying that only the square and its' content was set)